data/en/bots/425382319449309197.md

```diff
@@ -110,8 +115,9 @@ This permissions are:
 Just make sure to be on the guild and vote for the bot, to receive the special role.
 Future rewards may be added.
 
-# Useful Links
+# Usefull Links
```

Comment:
Pretty sure it's just `Useful`

Reply:
Already changed when you made that comment